kotlin/app/src/main/java/com/fullstorydev/shoppedemo/data/AppDatabase.kt

```diff
@@ -0,0 +1,35 @@
+package com.fullstorydev.shoppedemo.data
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+
+@Database(entities = [CartItem::class, Product::class], version = 1, exportSchema = false)
+abstract class AppDatabase : RoomDatabase() {
+    abstract fun productDao(): ProductDao
+    abstract fun itemDao(): CartItemDao
```

Comment:
nit: name consistency here
```suggestion
    abstract fun cartItemDao(): CartItemDao
```